StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -733,3 +743,15 @@ private func isEqual(_ lhs: STPPaymentIntentShippingDetails?, _ rhs: STPPaymentI
 
     return rhs == lhsConverted
 }
+
+private extension ConsumerPaymentDetails {
+    
+    var expectedPaymentMethodTypeForPassthroughMode: String? {
+        switch type {
+        case .card, .unparsable:
```

Comment:
Just checking, should we not send this for `.card`?

Reply:
As far as I know, that’s not necessary. Let me double-check!

Reply:
The backend will fall back to `card`, but I added it anyway since that’s what Web is doing 🤷‍♂️